src/main/java/forestry/Forestry.java

```diff
@@ -56,9 +60,10 @@
 		version = Constants.VERSION,
 		guiFactory = "forestry.core.config.ForestryGuiConfigFactory",
 		acceptedMinecraftVersions = "[1.12]",
-		dependencies = "required-after:forge@[14.21.1.2395,);"
+		dependencies = "required-after:forge@[14.21.1.2424,);"
```

Comment:
Please pick the minimum version of Forge that will work, 2424 doesn't seem to have an important change. 
Also add a comment on this line saying which new feature of Forge is required.